### DISCUSSION TITLE
`ADR Suggestion` Default branch naming

### DISCUSSION THREAD

**Discussion by AndrewSazonov** [easyscience/.github#6] [Ideas]
Since the beginning of GitHub, the default branch name was set to `master`. Some time ago, GitHub renamed the default branch from `master` to `main`. At some point in the [EasyScience organisation settings](https://github.com/organizations/EasyScience/settings/repository-defaults) we changed it back to `master`. 

As a result, some of our projects have default branches called `master` (e.g. EasyDiffraction, etc.) and others are called `main` (e.g. EasyReflectometry, etc.).

I propose to choose one or the other name and make it uniform for all repositories within the `EasyScience` organisation.

**Comment by rozyczko** [comment]
The reason it got switched to `main` in reflectometry was to fix a merge issue, where we saw backmerges not working properly.

I agree we should standardize on a single main repository and I also think it should be called `master`.

**Comment by AndrewSazonov** [comment]
So, the conclusion is to use `master` as the default branch name for the whole EasyScience organisation and to rename `main` to `master` in projects where this has not already been done.

**Comment by damskii9992** [comment]
Closing this ADR suggestion without making a proper ADR, as its content is covered in #12 .
